scipy/integrate/_bvp.py

```diff
@@ -357,12 +357,8 @@ def solve_newton(n, m, k, col_fun, bc, jac, y, p, B, tol, n_iter):
 
     Parameters
     ----------
-    n : int
-        Number of equations in the ODE system.
-    m : int
-        Number of nodes in the mesh.
-    k : int
-        Number of the unknown parameters.
+    n, m, k : int
+        Problem dimensions.
```

Comment:
For consistency considerations and to avoid repeating the full explanation in every function (i.e. use this short form everywhere). But as it only occurs in two functions currently I will change it to the more verbose version.